### PR TITLE
Add Contributers for display

### PR DESCRIPTION
**Pull request by DanielHabenicht**
Add them by referring to the [@all-contributers Bot](https://allcontributors.org/docs/en/bot/usage) in any pr or issue, or using the cli (`yarn all-contributers add <username> <emojikey>`).

The .all-contributerssrc should be consumable by the build to display all of them or we can add another markdown file and display it in the app. 

TODO:
- [x] Add the bot to the project https://allcontributors.org/docs/en/bot/installation @alexanderthieme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +42,35 @@ node dev/generate-random-data-for-geojson --help
...
+</table>
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
+
+> This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome! Add them by referring to the [@all-contributers Bot](https://allcontributors.org/docs/en/bot/usage) or using the cli (`yarn all-contributers add <username> <emojikey>`).
+
+
```
Comment (DanielHabenicht) [inline README.md:74]:
```suggestion
---
```

**Comment by kommander** [comment]
@DanielHabenicht ist das soweit mit dem bot?

**Comment by DanielHabenicht** [comment]
Ja

**Pull request merged**